libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -559,7 +564,7 @@ bool ofFile::openStream(Mode _mode, bool _binary){
 //------------------------------------------------------------------------------------------------------------
 bool ofFile::open(const of::filesystem::path & _path, Mode _mode, bool binary){
 	close();
-	myFile = ofToDataPath(_path);
+	myFile = ofToDataPath(_path, true);
```

Comment:
hmm this seems problematic - any reason it can't work relative?

Reply:
it can. most probably a residue of previous changes, will push another commit to revert back :)

Reply:
reverted and tested with fileUtils. 100%